packages/datepicker/src/__test__/use-date-picker.test.ts

```diff
@@ -91,13 +91,13 @@ describe('useDatePicker', () => {
       }),
     );
 
-    //Ensure that next/previous months buttons are disabled
+    //Ensure that next month button is disabled
     expect(result.current.propGetters.addOffset({ months: 1 }).disabled).toBe(
       true,
     );
     expect(
       result.current.propGetters.subtractOffset({ months: 1 }).disabled,
-    ).toBe(true);
+    ).toBe(undefined);
```

Comment:
The only thing I would like to see here is to get to the point when this become false. So you need to perform one extra click and maybe reload the state.

Reply:
I solved all the other comments, but I'm struggling with this one. I'm not sure if I understand your goal correctly and at the same time I may be not proficient enough in the lib functionality to be able to implement this. Please can you give some guidance on how to do this ?